packages/datalake-data-export/.env

```diff
@@ -0,0 +1,16 @@
+LOG_LEVEL=info
+
+READMODEL_DB_HOST=localhost
+READMODEL_DB_NAME=readmodel
+READMODEL_DB_USERNAME=root
+READMODEL_DB_PASSWORD=example
+READMODEL_DB_PORT=27017
+
+S3_BUCKET=interop-local-bucket
```

Comment:
🧹  Unused, you have it in `DATALAKE_STORAGE_BUCKET`

Reply:
Done in [c0856d5](https://github.com/pagopa/interop-be-monorepo/pull/973/commits/c0856d5690ed623c4b63ce0825f3c977bbf6a0e4)